sql/2020/19_Compression/type_of_content_encoding.sql

```diff
@@ -0,0 +1,18 @@
+#standardSQL
+#type of content_encoding
```

Comment:
There can be many mimeTypes. I wonder if these results should be filtered to exclude those with very few requests. For example:

```sql
HAVING
  num_requests > 1000
```

Reply:
nice point to clean very few rows, instead of re-filter manually. thanks.